src/lib/node.ts

```diff
@@ -230,13 +231,25 @@ export function pipeToNodeStream<T>(
  * @param nodeReadable Soource NodeJS stream
  * @returns Readable stream of type `string | Buffer`
  */
-export function fromNodeReadable(
+export function fromNodeReadable<T = void>(
```

Comment:
Hmm, why not just `export function fromNodeReadable<T = Buffer | string>(...): ReadableStream<T>` ?